### PR TITLE
Expose container stop and remove options in GenericContainer

### PR DESCRIPTION
**Pull request by cristianrgreco**
https://github.com/testcontainers/testcontainers-node/pull/31#discussion_r319225306

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,2 +1,3 @@
 export { GenericContainer } from "./generic-container";
+export { TestContainer, StartedTestContainer, StoppedTestContainer } from "./test-container";
```
Comment (markmsmith) [inline src/index.ts]: If you also export DEFAULT_STOP_OPTIONS then users can just override the parts they care about, eg
```ts
await testContainer.stop({
  ...DEFAULT_STOP_OPTIONS,
  timeout: 0
});
```

Comment (markmsmith) [reply src/index.ts]: Alternatively, you could do a merge between the passed options and the full set of options, which is probably nicer.

File: src/generic-container.ts [hunk 2]
```diff
@@ -132,9 +138,10 @@ class StartedGenericContainer implements StartedTestContainer {
     private readonly dockerClient: DockerClient
   ) {}
 
-  public async stop(): Promise<StoppedTestContainer> {
-    await this.container.stop();
-    await this.container.remove();
+  public async stop(options: OptionalStopOptions = {}): Promise<StoppedTestContainer> {
+    const resolvedOptions = { ...DEFAULT_STOP_OPTIONS, options };
```
Comment (markmsmith) [inline src/generic-container.ts]: I think you need a spread on `options` as well.

Comment (cristianrgreco) [reply src/generic-container.ts]: D'oh, thanks

**Pull request merged**